boards/arm/nrf51_pca10028/doc/nrf51_pca10028.rst

```diff
@@ -45,21 +45,33 @@ hardware features:
 +-----------+------------+----------------------+
 | Interface | Controller | Driver/Component     |
 +===========+============+======================+
-| NVIC      | on-chip    | nested vectored      |
-|           |            | interrupt controller |
 +-----------+------------+----------------------+
-| RTC       | on-chip    | system clock         |
-+-----------+------------+----------------------+
-| UART      | on-chip    | serial port          |
+| ADC       | on-chip    | adc                  |
 +-----------+------------+----------------------+
 | GPIO      | on-chip    | gpio                 |
 +-----------+------------+----------------------+
+| I2C       | on-chip    | i2c                  |
++-----------+------------+----------------------+
 | FLASH     | on-chip    | flash                |
 +-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
```

Comment:
There is no PWM hardware in nRF51.

Reply:
No, but we do support PWM as a feature on nRF51, via a software implementation using PPI.

Reply:
I know. But the table lists [hardware features](https://github.com/zephyrproject-rtos/zephyr/pull/10018/files#diff-cb524e164cf6545f4cb083364d79e721R43). Also the second column says that a controller is "on-chip". I think this may generate confusion.